pkg/epinio/detail/applications.vue

```diff
@@ -51,70 +49,141 @@ export default Vue.extend<Data, any, any, any>({
+      const matchGithub = str.match('^(https|git)(:\/\/|@)([^\/:]+)[\/:]([^\/:]+)\/(.+)(.git)*$');
+
+      return `${ matchGithub?.[4] }/${ matchGithub?.[5] }`;
     }
   },
+  computed: {
+    resourcesUssage() {
+      return this.value.instances?.reduce((acc: any, curr: { millicpus: number; memoryBytes: number }) => {
+        acc = {
+          millicpus:   acc.millicpus += curr.millicpus,
+          memoryBytes: acc.memoryBytes += curr.memoryBytes / (1024 * 1024) // bytes to megabytes
+        };
 
-  computed: {}
-
+        return acc;
+      }, {
+        millicpus:    0,
+        memoryBytes: 0,
+      });
+    },
+    choseIcon() {
+      return this.value.metadata.icon || 'fa-cube';
+    },
+    setIconType() {
+      switch (this.value.sourceInfo?.label) {
+      case 'File system':
+        return 'icon-file';
+      case 'Git':
+        return 'icon-github';
+      case 'Container':
+        return 'icon-docker';
+      default:
+        return 'icon-epinio';
+      }
+    },
+  }
 });
 </script>
 
 <template>
-  <div>
-    <div class="simple-box-row mt-40">
-      <SimpleBox class="routes">
-        <div class="box">
-          <h1>{{ value.routeCount }}</h1>
-          <h3>
-            {{ t('epinio.applications.detail.counts.routes') }}
-          </h3>
-        </div>
+  <div class="application-details">
+    <ApplicationCard>
+      <!-- Icon slot -->
+      <template v-slot:cardIcon>
+        <i class="icon icon-fw" :class="setIconType"></i>
+      </template>
+
+      <!-- Routes links slot -->
+      <template v-slot:top-left>
+        <h1>Routes</h1>
         <ul>
-          <li v-for="(route) in value.configuration.routes" :key="route.id">
+          <li v-for="route in value.configuration.routes" :key="route.id">
             <a v-if="value.state === 'running'" :key="route.id + 'a'" :href="`https://${route}`" target="_blank" rel="noopener noreferrer nofollow">{{ `https://${route}` }}</a>
-            <span v-else :key="route.id + 'a'">{{ `https://${route}` }}</span>
+            <span v-else :key="route.id + 'b'">{{ `https://${route}` }}</span>
           </li>
         </ul>
-      </SimpleBox>
-      <SimpleBox class="services">
-        <div class="box">
-          <h1>{{ value.configCount }}</h1>
-          <h3>
-            {{ t('epinio.applications.detail.counts.config') }}
-          </h3>
+      </template>
+
+      <template v-slot:top-right>
+        <!-- // TODO: Depends on https://github.com/epinio/epinio/issues/1345 -->
+        <!-- <span>Uptime 1d 23min</span>
+        Age: 2d ago -->
+      </template>
+
+      <!-- Resources count slot -->
+      <template v-slot:resourcesCount>
+        <!-- // TODO: Depends on https://github.com/epinio/epinio/issues/1471 -->
+        <!-- <div>
+          {{ value.configCount }} {{ t('epinio.applications.detail.counts.config') }}
+        </div> -->
+        <div>
+          {{ value.configCount }} {{ t('epinio.applications.detail.counts.services') }}
```

Comment:
There's `Configurations` and there's also `Services` (which are instances of `Catalog Services`). The app can be bound to both, however the `configurations` property currently contains resources that apply to both types (as part of binding a service to an application a configuration is counted). Until https://github.com/epinio/epinio/issues/1471 is created we should keep this labelled as `Configurations`